README.md

```diff
@@ -1,13 +1,103 @@
 # The PINK Annotation Schema
+The PINK Annotation Schema provides a semantic annotations for Safe and Sustainable by Design (SSbD) approach to guide the innovation process for chemicals and materials.
+It adhears to the recommendations specified by [DCAT-AP 3.0.1] and uses [PROV-O] for provenance.
 
-The PINK Annotation Schema provides a semantic annotations for Safe and Sustainable by Design (SSbD) approach to guide the innovation process for chemicals and materials. It adhears to the recommendations specified by [DCAT-AP 3.0.1].
-
-[DCAT-AP 3.0.1]: https://semiceu.github.io/DCAT-AP/releases/3.0.1/
 
 ## Repository Files
-
 - `catalog-v001.xml`: XML catalog mapping ontology files to their IRIs for semantic web tools.
 - `contributors.ttl`: Turtle file listing contributors to the PINK Annotation Schema for this repository.
 - `pink_annotation_schema.ttl`: Main ontology file for the PINK Annotation Schema.
 - `reused-terms.ttl`: Terms from standard vocabularies reused by the schema.
-- `LICENSE`: Creative Commons Attribution 4.0 International license
+- `LICENSE`: Creative Commons Attribution 4.0 International license.
+- `docs/`: Sub-directory with documentation.
+
+
+## Taxonomy
+The taxonomy below shows a basic categorisation of the main concepts (OWL classes) in PINK Annotation Schema.
+It unifies concepts from common vocabularies, like [Dublin Core], [DCAT], [PROV-O] and [FOAF].
+This gives the adapted terms additional context.
+However, the taxonomy is still intentionally weekly axiomated in order to make it easy to align it to different popular top-level ontologies, like EMMO, DOLCE and BFO.
+
+
+![Taxonomy](docs/figs/taxonomy.png)
+
+At the top-level, the PINK Annotation Schema has four root concepts:
+
+- **`prov:Entity`**: A physical, digital, conceptual, or other kind of thing with some fixed aspects.
+  [PROV-O] lacks the accuracy of nominalism and allows entities may be real or imaginary.
+
+- **`prov:Activity`**: Something that occurs over a period of time and acts upon or with entities.
+  Hence, its individuals have some temporal parts (that are not of the same type as the activity itself).
+  An activity may include consuming, processing, transforming, modifying, relocating, using, or generating entities.
+
+- **`:Role`**: The class of all individuals that are defined through a parthood relation to an entity.
+  The individual can be said to have a *role* in relation to the entity.
+
+- **`foaf:Agent`**: A thing that do stuff (like person, group, software or physical artifact).
+  The [FOAF] specification of a *agent* is very loose.
+  The subclass `prov:Agent` provides further context, but saying that a `prov:agent` has a role in an activity.
```

Comment:
```suggestion
  The subclass `prov:Agent` provides further context, by saying that a `prov:agent` bears some form of responsibility for an activity, the existense of an entity or the activity of another agent-
```

Reply:
I am not sure I agree with what is written about role. Can we have a talk about it?